.wp-env.json

```diff
@@ -6,6 +6,7 @@
     "./plugins/dominant-color-images",
     "./plugins/embed-optimizer",
     "./plugins/optimization-detective",
+    "./plugins/performance-lab",
```

Comment:
I think the first line, `".",` can be removed.

Reply:
That is required to mount the root files in the container.

Reply:
Ah ok, is that so you can run unit tests from inside the container, or are there other reasons?

One of the side effects of adding that line to the `plugins` property of the `wp-env` object is that it will try to find and activate a plugin at the root, which I think we want to avoid. We might be better off using the `mappings` property to place any extra folders we need into the plugin directory root, or optionally, only do this in the test environment where they're needed, e.g.:

```
"env": {
        "tests": {
            "mappings": {
                "/wp-content/plugins/": "."
            },
        }
    }
```

Reply:
> Ah ok, is that so you can run unit tests from inside the container, or are there other reasons?

Yes, that's correct. Besides that, I don't see any other reasons.

Thanks for suggesting mappings. It's pointless to activate the base plugin as it only adds a confusing notice. I'll give it a try and hopefully, it will resolve this issue.

Reply:
> The other thing I noticed while trying to test the unit tests with this setup is that we will need to remove the $(basename $(pwd)) part of each of the npm scripts for running unit tests now that we're doing so from the plugin root and not from the performance lab plugin root.

We have an ongoing discussion on this topic at https://github.com/WordPress/performance/issues/1012. Perhaps we should address this improvement as part of that ticket? Currently, all our tooling is configured to run from the monorepo plugin root.

Reply:
Fixed in https://github.com/WordPress/performance/pull/1182/commits/23de67ac27966a97dc752e95ccd0c18eef5423db and https://github.com/WordPress/performance/pull/1182/commits/db957d47717f978c5c0673e6a518b5762d56ef19